src/mono/mono/mini/exceptions-amd64.c

```diff
@@ -1413,7 +1413,7 @@ mono_arch_unwindinfo_insert_range_in_table (const gpointer code_block, gsize blo
 			new_entry->handle = NULL;
 			new_entry->begin_range = begin_range;
 			new_entry->end_range = end_range;
-			new_entry->rt_funcs_max_count = (block_size / MONO_UNWIND_INFO_RT_FUNC_SIZE) + 1;
+			new_entry->rt_funcs_max_count = (DWORD)((block_size / MONO_UNWIND_INFO_RT_FUNC_SIZE) + 1);
```

Comment:
Instead of `DWORD` can you try `uint32_t` instead?

Reply:
I used the types (aliases), which compiler was expecting in a given context. Generally, mono uses `g<type>` types. Note that compiler warns with the same type name which the given expression is expected to have (e.g. which the function prototype has)

What do you think, shall we keep it?

Reply:
No, you're right here. Let's keep it unless the Mono folks push back.

Reply:
This is Windows specific code and the underlying struct is using DWORD for this field since it is used with Win32 functions assuming DWORD's, so I think the cast is fine.